### DISCUSSION TITLE
How to make MiKTex potable

### DISCUSSION THREAD

**Discussion by lylgithub2021** [MiKTeX/miktex#840] [Q&A]
Following the instruction on MiKTex website(see https://miktex.org/howto/portable-edition), Ｉ installed the latest portable version of MiKTex(basic-miktex-21.2-x64.exe) and it works well. Then I move the installed MiKTex folder into another place which cause everything changed, and the MiKtex can not work any more. This means it is not portable. 
So, in my case, how to make the MiKTex portable that can be placed everywhere.
Thanks.

**Comment by spectorsky** [comment]
What error message (among others) are you getting now? 
At least, you have to add your bin directory to path environment. 
Maybe it worths launching MiKTeX console and check whether your root and (if created) local directories are set correctly.
